subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1441,39 +1491,47 @@ int nrf_cloud_pgps_process(const char *buf, size_t buf_len)
 	if (index.store_block == NO_BLOCK) {
 		LOG_ERR("No free flash space!");
 		state = PGPS_NONE;
+		npgps_unlock();
 		return -ENOMEM;
 	}
 	index.storage_extent = npgps_get_block_extent(index.store_block);
-	LOG_INF("opening storage at block:%d, len:%d", index.store_block,
+	LOG_DBG("opening storage at block:%d, len:%d", index.store_block,
 		index.storage_extent);
 	err = open_storage(npgps_block_to_offset(index.store_block),
 			   index.partial_request);
 	if (err) {
 		state = PGPS_NONE;
+		npgps_unlock();
 		return err;
 	}
 
 	index.dl_offset = 0;
 	ignore_packets = true;
-	int sec_tag = SEC_TAG;
+	LOG_DBG("Ready for updated P-GPS data");
 
-	if (FORCE_HTTP_DL && (strncmp(pgps_dl.host, "https", 5) == 0)) {
-		int i;
-		int len = strlen(pgps_dl.host);
+	return 0;
+}
 
-		for (i = 4; i < len; i++) {
-			pgps_dl.host[i] = pgps_dl.host[i + 1];
-		}
-		sec_tag = -1;
-	}
+#if !defined(CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_CUSTOM)
```

Comment:
Do you mind adding a few hints in key places (here, maybe the function descriptions, etc) that the HTTP download transport calls the the start_update/finish_update functions internally?